source/class/cv/ui/structure/tile/components/Color.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Shows a html color input
+ * @author Tobias Bräutigam
+ * @since 2022
+ */
+qx.Class.define('cv.ui.structure.tile.components.Color', {
```

Comment:
Due to the (Professional) ColorChooser the CV has a quite thorough color handling class. Probably it should be used here - especially as the visu color input will most likely be used for lighting applications where the basic web assumption "everything is sRGB" doesn't hold anymore

Reply:
Please ignore the current state of the color component, its not even a proof-of-concept more a proof-that-the-html-color-input-is-no-solution-for-this. I haven't decided yet which way I want ot go with the color-component here. but I do not want to have the complexity of the existing ColorChooser here. I want something very simple, which will fit my needs and probably the needs of must of the users. Might be an option to implement a way to integrate the professional colorchooser for users the need more features, but thats something for "later".